src/estimagic/optimization/subsolvers/bounded_newton_quadratic.py

```diff
@@ -252,19 +304,19 @@ def compute_predicted_reduction_from_conjugate_gradient_step(
     active_bounds_info,
 ):
     """Compute predicted reduction induced by the Conjugate Gradient step."""
-    if active_bounds_info.all.size > 0:
+    if active_bounds_info.active.size > 0:
```

Comment:
Was this a renaming or an actual bugfix?

Reply:
``active_bounds_info`` is of type ``class(NamedTuple)``. I threw an error moving from ``namedtuple`` to ``NamedTuple``, as``all`` appears to be an internal keyword argument for ``class(NamedTuple)``.

Reply:
ok. Thanks!